twisted/python/logfile.py

```diff
@@ -62,11 +62,12 @@ def _openFile(self):
         Open the log file.
 
         We don't open files in binary mode since:
-        * an encoding would have to be chosen and that would have to be
-          configurable
-        * Twisted doesn't actually support logging non-ASCII messages
-          (see U{https://twistedmatrix.com/trac/ticket/989})
-        * logging plain ASCII messages is fine with any non-binary mode.
+
```

Comment:
http://epydoc.sourceforge.net/epytext.html . Please see section 3.1 for unordered list